common/lib/capa/capa/responsetypes.py

```diff
@@ -380,7 +388,12 @@ def make_hint_div(self, hint_node, correct, student_answer, question_tag,
             style = QUESTION_HINT_INCORRECT_STYLE
 
         # Ready to go
-        return u'<div class="{0}">{1}{2}</div>'.format(style, label_wrap, hints_wrap)
+        return u'<div class="{style}"><div class="explanation-title">{text}:</div>{labelwrap}{hintswrap}</div>'.format(
```

Comment:
Concatenating ":" on to text won't work-- it won't translate well, as colon doesn't make sense in all languages. It will need to be part of the text itself.